src/test/java/de/hbt/power/service/SkillControllerTest.java

```diff
@@ -126,5 +126,61 @@ public void testUpdateAndGetCategoryWithPriorExistence() throws Exception {
         }
     }
 
+    /**
+     * Tests whether a skill that didn't have a category before will be categorized
+     * @throws Exception
+     */
+    @Test
+    public void testUpdateAndGetCategory_WithoutCategory_ShouldHaveCategory() throws Exception {
+        for (String skillName : skillNames) {
+            Skill skill = new Skill();
+            skill.setQualifier(skillName);
+            skill.setCategory(null);
+            skillRepository.save(skill);
+            MockHttpServletRequestBuilder builder = post("/skill");
+            builder.param("qualifier", skillName);
+            mockMvc.perform(builder).andExpect(status().isOk());
+            SkillCategory category = skillRepository.findOneByQualifier(skillName).map(Skill::getCategory).orElse(null);
+            assertThat(category != null);
```

Comment:
Siehe unten
```suggestion
             assertThat(category).isNotNull();
```
https://joel-costigliola.github.io/assertj/assertj-core-quick-start.html